interface/therapy_groups/therapy_groups_views/groupDetailsParticipants.php

```diff
@@ -0,0 +1,307 @@
+                                        </div>
+                                    </div>
+                                </form>
+                            </div>
+                            <div class="row">
+                                <div class="col-md-12">
+                                    <form id="updateParticipants" method="post">
+                                        <input type="hidden" name="group_id" value="<?php echo attr($groupId); ?>" />
+                                        <button id="addParticipant"><?php echo xlt('Add'); ?></button>
+                                        <table  id="participants_table" class="dataTable display">
+                                            <thead>
+                                            <tr>
+                                                <th><?php echo xlt("Participant's name"); ?></th>
+                                                <th><?php echo xlt("Patient's number"); ?></th>
+                                                <th><?php echo xlt('Status in the group'); ?></th>
+                                                <th><?php echo xlt('Date of registration'); ?></th>
+                                                <th><?php echo xlt('Date of exit'); ?></th>
+                                                <th><?php echo xlt('Comment'); ?></th>
+                                                <?php if($readonly == ''): ?>
+                                                    <th><?php echo xlt('Delete'); ?></th>
+                                                <?php endif; ?>
+                                            </tr>
+                                            </thead>
+                                            <tbody>
+                                            <?php foreach ($participants as $i => $participant) : ?>
+                                                <tr>
+                                                    <td>
+                                                        <input type="hidden" name="pid[]" value="<?php echo htmlspecialchars($participant['pid'],ENT_QUOTES); ?>" />
+                                                        <span><?php echo text($participant['lname']) .', ' . text($participant['fname']); ?></span>
+                                                    </td>
+                                                    <td><span><?php echo text($participant['pid']); ?></span></td>
+                                                    <td>
+                                                        <select name="group_patient_status[]" <?php echo $readonly; ?>>
+                                                            <?php foreach ($statuses as $key => $status): ?>
+                                                                <option value="<?php echo attr($key);?>" <?php if($key == $participant['group_patient_status']) echo 'selected'; ?> > <?php echo xlt($status); ?> </option>
```

Comment:
Isn't status already translated now. If so, then don't need to translate it here.